README.adoc

```diff
@@ -60,3 +60,7 @@ spring.cloud.gcp.credentials-location=file:[LOCAL_PRIVATE_KEY_FILE]
 These properties are optional and, if not specified, Spring Boot will attempt to automatically find
 them for you. For details on how Spring Boot finds these properties, refer to the desired starter
 README.
+
+If your app is running on Google App Engine or Google Compute Engine, you should omit the
+`spring.cloud.gcp.credentials-location` property and, instead, let the Spring Cloud GCP Core Starter
```

Comment:
Why should we say "you should omit" meanwhile it would be better to ignore `spring.cloud.gcp.credentials-location` property in case of those environments (some `@Conditional` ?) and even WARN end-user that it does not make sense.

WDYT?

Reply:
When would you warn users? If you try to deploy an app with a credentials path on your local file system, the deployment just fails.

We might want to add default credentials on our App Engine conditional, for example. But we would be overriding a user property, which I'd rather not. And that wouldn't clear the need to document this, anyway.

Reply:
I would just say "If your app is running on Google App Engine or Google Compute Engine, in most cases, you will want to omit the `spring.cloud.gcp.credentials-location` property and rely on the built-in credentials."